frontend/app/routes/protected/person-case/primary-docs.tsx

```diff
@@ -1,7 +1,7 @@
-import { useId } from 'react';
+import { useId, useState } from 'react';
 
 import { data, useFetcher } from 'react-router';
-import type { RouteHandle } from 'react-router';
+import type { RouteHandle, SessionData } from 'react-router';
```

Comment:
Import `SessionData` from `express-session`. We are augmenting `SessionData` from `express-session` and not `react-router`. Data type of `PrimaryDocumentsSessionData` is `any` at the moment.